### ISSUE TITLE
Bug with layout of segment filters

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#13799]
### Mautic Version

5.0.x series

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8

### What happened?

I noticed when reviewing another PR that there's a small bug with the layout of the segment filters when viewing in list view:

![mobile-segment](https://github.com/mautic/mautic/assets/2930593/fd0863fd-6e32-4616-8e16-79c741426917)

I think that this should be stacked one above the other and should look visually the same as the search filter above rather than having the black border.

cc @mautic/ux-ui-tiger-team 

### How can we reproduce this issue?

Step 1: Open 5.x branch and view in mobile resolution
Step 2: Check the segments list and look at the filters above the list

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Note: Also noticed the same issue on campaigns:

![screenshot-mautic ddev site-2024 05 31-12_40_56](https://github.com/mautic/mautic/assets/2930593/373c9429-36bc-4484-b15e-5cc0decb250c)

Also on emails:

![screenshot-mautic ddev site-2024 05 31-12_41_50](https://github.com/mautic/mautic/assets/2930593/6c75175f-4ec5-4e2b-8a01-932422f3bf86)

Also on categories:

![screenshot-mautic ddev site-2024 05 31-12_43_08](https://github.com/mautic/mautic/assets/2930593/c21023f7-6bcc-45c9-aea2-1d0b04235add)

**Comment by obamwonyi** [comment]
Hi, is this issue still open? it seems to have been assigned.

**Comment by andersonjeccel** [comment]
@obamwonyi Still open if you'd like to fix it

**Comment by RCheesley** [comment]
This is now resolved with the new user interface.
